client/index.js

```diff
@@ -8,10 +8,17 @@ import thunk from 'redux-thunk'
 import reducers from './reducers'
 import App from './components/App'
 
-const store = createStore(reducers, compose(
-  applyMiddleware(thunk),
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
-))
+let store
+if (window.__REDUX_DEVTOOLS_EXTENSION__) {
+  store = createStore(reducers, compose(
+    applyMiddleware(thunk),
+     window.__REDUX_DEVTOOLS_EXTENSION__()
+  ))
+} else {
+  store = createStore(reducers, compose(
+    applyMiddleware(thunk)
+  ))
+}
```

Comment:
According to https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup, it could look like this:

```
  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
  const store = createStore(reducer, composeEnhancers(
    applyMiddleware(...middleware)
  ))
```

A bit cleaner ;)